### PR TITLE
Update sanity check to cover .github and .vscode folders

### PR DESCRIPTION
**Pull request by reyang**
Follow up #3556, https://github.com/open-telemetry/opentelemetry-dotnet/pull/3548#discussion_r939173638 and https://github.com/open-telemetry/opentelemetry-dotnet/pull/3548#discussion_r938328292.

### CODE REVIEW

**Review by utpilla — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -49,12 +49,12 @@ jobs:
     - name: Autobuild
       uses: github/codeql-action/autobuild@v2
 
-    # ℹ️ Command-line programs to run using the OS shell.
-    # 📚 https://git.io/JvXDl
```
Comment (alanwest) [inline .github/workflows/codeql-analysis.yml:53]: but but... I like emojis. Can we have sanitycheck require minimum one emoji per file?

**Review by alanwest — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (reyang) [reply .github/workflows/codeql-analysis.yml:53]: Is there an emoji for OpenTelemetry (maybe 🔭)? Might be a good thing (although it's not friendly to developers who use Notepad.exe or vi via telnet) 😃

Comment (alanwest) [reply .github/workflows/codeql-analysis.yml:53]: >  (although it's not friendly to developers who use Notepad.exe or vi via telnet)

I have a solution:

```
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWX0KNWMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWX0OkkOKNWMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWX0OkkkkkkOKNWMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMNKOkkkkkkkkkkOKNWMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMN0kkkkkkkkkkkkkOKNWMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWNKOkkkkkkkkkkkkkOKNWMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWNKOkkkkkkkkkkkkkOKNWMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWXOkOXWMWNKOkkkkkkkkkkkkkOKNWMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWXkl:;:cxKWMWNKOkkkkkkkkkkkkkOKNWMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWWXkl:;;:;;:lxKWMWNKOkkkkkkkkkkkkkOKNWM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWXkdl:;:;;;;;;;:cxKWMWNKOkkkkkkkkkkkkkOXW
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWXkl:;;;;:;;;;;;;;::cxKWMWNKOkkkkkkkkkkkOKW
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMWXkl:::;;;;::;;;:;;:;;;:cxKWMWNKOkkkkkkkO0XWM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMKo:;;;;:;;:;;;;;;;;;::;;;:cxKWMWNKOkkkO0XWMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMKo:;;;;;;;;;;;;;;;;:;;:;;:;:ckNMMWNKKKXWMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMNOl:;;::;;;;;;;;;;;;;;;;;;;:dXMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMNOo:::;;::ccllcc:;:;;:;:lkXWMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMWKkdx0NMMXOxooxO0XXNNXK0kdl::lkXWMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMW0dc:;;:d0NMMWWWWMMWWWWWMMMWXOOXWMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMW0dc;::;;;:cOWMMMMWXK00OO0KXNWMMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMW0dc:;;;;;;;:oKMMMWX0kkkOOOOkkOKNMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMW0dc:;;;;;;;;;l0WMWNKOkk0XNWNNKOkk0NMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMW0dc;;;;;;;;;;;;oXMMNK0kk0NMMMMMWKkkOKWMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMW0dc;;;;;:;;;;;;;;oXMMNX0kk0NMMMMMWKkkOXWMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMXxc:;;:;;:;;;:;;;;;c0WMWWXOkkOKXNNX0OkkKNMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMM0l:;;;;;::;::;;;;;:ckNMMMWX0OkkkOOkkkOKNMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMWKdc:;;:;;;;;;;;;cd0NMMMMMMWNXK0000KXWWMMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMWKdc:;;:;;:;:cd0NMMMMMMMMMMMMWWWWMMMMMMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMWWXKXNWMWKdc:;;;;cd0NMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMWNK0kkkOKNWMWKxc:cd0NMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
MNKKXNNX0kkkkkkk0NMMMWXKKWMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
WXOkkOOkkkkkkkkOXWMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
MWX0OkkkkkkkkOKNWMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
MMMWX0OkkkkOKNWMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
MMMMMWX0OkkOKWMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMWX0OkOXMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMWNXNWMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM
```

Comment (reyang) [reply .github/workflows/codeql-analysis.yml:53]: Added more 💰💰💰:

```
 $$$$$$\                             $$$$$$$$\        $$\                                    $$\                         
$$  __$$\                            \__$$  __|       $$ |                                   $$ |                        
$$ /  $$ | $$$$$$\   $$$$$$\  $$$$$$$\  $$ | $$$$$$\  $$ | $$$$$$\  $$$$$$\$$$$\   $$$$$$\ $$$$$$\    $$$$$$\  $$\   $$\ 
$$ |  $$ |$$  __$$\ $$  __$$\ $$  __$$\ $$ |$$  __$$\ $$ |$$  __$$\ $$  _$$  _$$\ $$  __$$\\_$$  _|  $$  __$$\ $$ |  $$ |
$$ |  $$ |$$ /  $$ |$$$$$$$$ |$$ |  $$ |$$ |$$$$$$$$ |$$ |$$$$$$$$ |$$ / $$ / $$ |$$$$$$$$ | $$ |    $$ |  \__|$$ |  $$ |
$$ |  $$ |$$ |  $$ |$$   ____|$$ |  $$ |$$ |$$   ____|$$ |$$   ____|$$ | $$ | $$ |$$   ____| $$ |$$\ $$ |      $$ |  $$ |
 $$$$$$  |$$$$$$$  |\$$$$$$$\ $$ |  $$ |$$ |\$$$$$$$\ $$ |\$$$$$$$\ $$ | $$ | $$ |\$$$$$$$\  \$$$$  |$$ |      \$$$$$$$ |
 \______/ $$  ____/  \_______|\__|  \__|\__| \_______|\__| \_______|\__| \__| \__| \_______|  \____/ \__|       \____$$ |
          $$ |                                                                                                 $$\   $$ |
          $$ |                                                                                                 \$$$$$$  |
          \__|                                                                                                  \______/ 
```

Comment (TylerHelmuth) [reply .github/workflows/codeql-analysis.yml:53]: The beauty of the install that is the [opentelemetry-demo chart](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-demo/templates/NOTES.txt)

```
=======================================================================================


 ██████╗ ████████╗███████╗██╗         ██████╗ ███████╗███╗   ███╗ ██████╗ 
██╔═══██╗╚══██╔══╝██╔════╝██║         ██╔══██╗██╔════╝████╗ ████║██╔═══██╗
██║   ██║   ██║   █████╗  ██║         ██║  ██║█████╗  ██╔████╔██║██║   ██║
██║   ██║   ██║   ██╔══╝  ██║         ██║  ██║██╔══╝  ██║╚██╔╝██║██║   ██║
╚██████╔╝   ██║   ███████╗███████╗    ██████╔╝███████╗██║ ╚═╝ ██║╚██████╔╝
 ╚═════╝    ╚═╝   ╚══════╝╚══════╝    ╚═════╝ ╚══════╝╚═╝     ╚═╝ ╚═════╝ 
                                                                          
```

**Pull request merged**